README.md

```diff
@@ -158,4 +158,4 @@ This uses [BlueRaja's](https://github.com/BlueRaja/High-Speed-Priority-Queue-for
 I can't guarantee that this is bug-free, or that it will necessarily work for you. I've done my best to test and optimize this, but it is far from perfect. If you find any bugs, report them here on Github by opening an issue.
 
 ## Contribution
-Any contiribution is welcome. If you make any changes that you think other people would benefit from, create a Pull Request and I'll merge it as soon as possible. 
+Any contributions is welcome. If you make any changes that you think other people would benefit from, create a Pull Request and I'll merge it as soon as possible. 
```

Comment:
I think that was already correct. It means:
_Any (single) contribution is welcome._

If it were to be changed, it should probably be changed to:
_Any contributions **are** welcome._